tictactoe/tictactoe.js

```diff
@@ -0,0 +1,94 @@
+var origBoard;
+var player;
+const player1   = "X";
+const player2   = "O";
+const winCombos = [
+    [0,1,2],
+    [3,4,5],
+    [6,7,8],
+    [0,3,6],
+    [1,4,7],
+    [2,5,8],
+    [0,4,8],
+    [2,4,6],
+]
+const cells = document.querySelectorAll(".cell");
+startGame();
+
+function startGame() {
+    document.querySelector("#endgame").style.display = "none";
+    document.querySelector("#restart").style.display = "none";
+    player = player1 ;
+    document.getElementById("player").innerHTML = "X" ;
```

Comment:
The value should be `player` instead of `"X"`, right?

Reply:
Not here. This tells the program to show that it's X's turn at the start and when the game is reset.